### PR TITLE
LPD-20059 Create a Draft Return Request from Placed Order’s Details page

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**
https://liferay.atlassian.net/browse/LPD-20059

~~Depending on https://github.com/liferay-commerce/liferay-portal/pull/4775~~

/cc @andrea-ale-sbarra @alessiorendina

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Review by luca-pellizzon — COMMENTED** [review]
I think the breaking change description should be in the commit that is introducing the change not the one with the versioning.

Code looks good, just a question regarding one endpoint

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java [hunk 1]
```diff
@@ -441,6 +447,22 @@ public String getCommercePriceDisplayType() {
 		return commerceChannel.getPriceDisplayType();
 	}
 
+	public String getCommerceReturnableItemsAPIURL() {
+		return StringBundler.concat(
+			"/o/headless-commerce-delivery-order/v1.0/placed-orders/",
+			getCommerceOrderId(), "/placed-order-items");
```
[the remark this answers is not in the document]
Comment (gianmarcobrunialti) [reply modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java:453]: Correct. 👍🏼

**Comment by gianmarcobrunialti** [comment]
> I think the breaking change description should be in the commit that is introducing the change not the one with the versioning.
> 
> Code looks good, just a question regarding one endpoint

Thanks @luca-pellizzon ! I followed other examples for the breaking change commit description and it seemed to have been on Semantic versioning, e.g. debb56d676c90996cbf1e7f5bc96c40d59f9dafb, or 8306a26fc074f031cdd4578f1d0476b1f2fb0468, or 30fc0594f345bd106fe71e8f18380855be661449 to name a few, while in other cases it's on the change itself. So as you prefer 👍🏼

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
see https://github.com/brianchandotcom/liferay-portal/pull/148817

**Pull request closed without merging**